metrics/ci_integrations/lib/coverage/coverage_json_summary/model/coverage.dart

```diff
@@ -5,7 +5,7 @@ import 'package:metrics_core/metrics_core.dart';
 /// The class that represents the code coverage entity.
 class Coverage extends Equatable {
   /// The code coverage percent.
-  final Percent percent;
+  final PercentValueObject percent;
```

Comment:
```suggestion
  final Percent percent;
```
as this is a simple object that used widely, it feels like we can simplify reading.